isofit/radiative_transfer/radiative_transfer_engine.py

```diff
@@ -306,6 +309,7 @@ def build_interpolators(self):
                 lut_interp_types=self.lut_interp_types,
                 version=self.interpolator_style,
             )
+        del self.lut
```

Comment:
Deleting `self.lut` causes downstream issues with the properties:
```python
coszen = property(lambda self: self["coszen"])
solar_irr = property(lambda self: self["solar_irr"])
```
It could be worthwhile to modify the `RTE.__init__` to save certain variables from the lut file to the class object so we can delete `self.lut`

Reply:
This is the reason the examples are failing at the moment